tests/post_training/test_templates/test_quantizer_config.py

```diff
@@ -85,10 +85,27 @@ class TestGetStatisticsCollectorParameters:
         ref_per_ch_reduction_axes: List[int]
         ref_per_tensor_reduction_axes: List[int]
 
-    @abstractmethod
-    @pytest.fixture
+    @pytest.fixture(
+        params=[
+            pytest.param(
+                TestGetStatisticsCollectorParameters(TargetType.PRE_LAYER_OPERATION, "/Sum_1_0", (2,), (1, 2)),
```

Comment:
Was the `102414` ticket solved? Why did you missed axes?

Reply:
It was a skipped testcase I unskipped it and it works. Looks like the initial issue was resolved.
@daniil-lyakhov could you confirm?

Reply:
Confirm https://github.com/openvinotoolkit/nncf/pull/1607